driver/src/driver/stablex_driver.rs

```diff
@@ -154,9 +154,12 @@ impl<'a> StableXDriver for StableXDriverImpl<'a> {
             Err(err) => return DriverResult::Retry(err),
         };
 
+        // Make sure the solver has at least some minimal time to run to have a chance for a
+        // solution. This also fixes an assert where the solver fails if the timelimit gets rounded
+        // to 0.
```

Comment:
Where is the rounding being done?

Reply:
https://github.com/gnosis/dex-services/blob/2f87ca88761d3e97aad566939d9b67f7ff91b9ae/driver/src/price_finding/optimization_price_finder.rs#L290